app/Cart.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App;
+
+class Cart
+{
+    public $items = null;
+    public $totalQty = 0;
+    public $totalPrice = 0;
+
+    public function __construct($oldCart) {
+        if ($oldCart) {
```

Comment:
Mục đích cái này để làm gì thế?
Thấy mỗi hàm lại phải check item có tồn tại hay ko

Reply:
$oldCart = session()->has('cart') ? session()->get('cart') : null;
$cart = new Cart($oldCart);
Trường hợp cart đã tồn tại rồi thì khi tạo 1 class Cart gán các giá trị của cart đã tồn tại lại rồi mới tính toán tiếp